ansible/roles/linux-data-volumes/files/data-volumes-configure-volume.yml

```diff
@@ -9,6 +9,15 @@
   set_fact:
     data_volume_id: "{{ openstack_metadata[data_volume.metadata_var] }}"
 
+- name: Get Azimuth user metadata
+  set_fact:
+    azimuth_username: "{{ openstack_metadata['azimuth_workstation_username'] }}"
```

Comment:
I think we should hardcode this to azimuth for now, as we haven't fixed the other things looking to use the user called "azimuth".

Reply:
Done